app/src/main/java/com/mk/tasky/agenda/home/presentation/components/HomeDayPickerItem.kt

```diff
@@ -0,0 +1,60 @@
+package com.mk.tasky.agenda.home.presentation.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.clickable
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.mk.tasky.ui.theme.Black
+import com.mk.tasky.ui.theme.Gray
+import com.mk.tasky.ui.theme.Orange
+import java.time.LocalDateTime
+
+@Composable
+fun HomeDayPickerItem(
+    day: LocalDateTime,
+    isSelected: Boolean = false,
+    onDayClick: () -> Unit,
+    modifier: Modifier = Modifier
+) {
+    Column(
+        modifier = modifier.clip(RoundedCornerShape(100))
+            .background(if (isSelected) Orange else Color.Transparent).padding(8.dp).clickable {
```

Comment:
If you apply the clickable modifier before the padding, the click ripple effect will match the shape :)

Reply:
Right! Fixed